combat-trainer.lic

```diff
@@ -862,11 +862,7 @@ class LootProcess
       game_state.undissectable(mob_noun)
       return false
     when 'would probably object', "should be left alone."
-      fput ('dissect')
-    when "That's going to be hard"
```

Comment:
Should `"That's going to be hard"` no longer be in the bput matches on line 856?

Reply:
Removed. Let's find out if we need it or not! :)